node/send_http_request/index.js

```diff
@@ -0,0 +1,37 @@
+const axios = require("axios");
+module.exports = async (req, res) => {
+  let url, method, body, headers;
+
+  try {
+    const payload = JSON.parse(req.payload);
+    url = payload.url;
+    method = payload.method || undefined;
+    headers = payload.headers || undefined;
+    body = payload.body || undefined;
+  } catch (err) {
+    console.log(err);
+    throw new Error("Payload is invalid.");
+  }
+
+  const config = {
+    method,
+    url,
+    data: body,
+    headers,
+  };
+
+  try {
+    const response = await axios(config); // Request to Server
+
+    res.json({
+      success: true,
+      response: {
+        headers: response.headers,
+        code: response.status,
+        body: response.data,
+      },
+    });
+  } catch (error) {
+    res.json({ success: false, message: "URL could not be reached." });
```

Comment:
Let's add `console.log` here just like in try/catch earlier, for consistency.

Reply:
As I asked Steven, He said to remove the `cosole.log` because it was causing issues in running (with docker not with appwrite CLI). So I have removed all `console.log`  now